cartoframes/viz/layer.py

```diff
@@ -71,7 +69,7 @@ class Layer(object):
                     'hover': '$name',
                     'click': ['$name', '$pop_max', '$pop_min']
                 },
-                legend={
+                legends={
```

Comment:
This `dict` approach is no longer supported, right?

Reply:
It's not, but I'll change the Layer docs in the Style issue, so I didn't considered doing it here to avoid duplicating work 👍